src/main/g8/default.properties

```
@@ -1,5 +1,4 @@
 name=zio-quickstart
 description=This is a seed project that creates Scala 3 based ZIO application using zio-http, protoquill, caliban etc.
-dotty_version=3.1.3
-add_http_endpoint=yes
-package=com.example
\ No newline at end of file
+package=com.example
+scala3_version=3.2.1
```

```suggestion
scala_version=3.2.1
```

to be consistent with the other template